src/app/Exceptions/BackpackProRequiredException.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+namespace Backpack\CRUD\app\Exceptions;
+
+class BackpackProRequiredException extends \Exception
+{
+    /**
+     * Render the exception into an HTTP response.
+     *
+     * @param  \Illuminate\Http\Request
+     * @return \Illuminate\Http\Response
+     */
+    public function render($request)
+    {
+        // 0, by default, pass only what is a feature we construct the rest of the message
+        // 1 use the provided message in full
+        switch ($this->getCode()) {
+            case 0:
+                $this->message = $this->message.' is a Backpack PRO feature. Please purchase and install <a href="https://backpackforlaravel.com/pricing">Backpack\PRO</a>.';
+                break;
+        }
+
+        return response(view('errors.500', ['exception' => $this]), 500);
```

Comment:
Alternatively, instead of throwing a general 500, we could load a custom error view, that lives inside the Backpack repo (eg. `backpack::backpack_error`)... Then over there, we could have additional stuff like buttons after the error message... or something. We could also customize the look & feel of this page, to stand out and provide more information, since normally it's the developer who sees it.

Alternatively, we could maybe throw a Flare error, but just with our custom error message...

What do you think @pxpm ?

Reply:
I think for now we can just go ahead and remove the links. Then later on, when we have more time, we can create a cool error view, that allows you to pass more information like buttons and such. We'll think of what exactly an exception page needs then.

Reply:
I agree with you. I think it would be WAY WAY more usefull in this cases where the problem is an "Application setup problem". We can have extra links, we can even have a "small" api on that page that could fetch the "articles/faq" that contain the keywords of the reported error.

It would also clearly differentiate from a server error (500) and a Backpack setup error. 

Hope we do it! 🙏